definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+                set -o pipefail
+                set -o errexit
+
+                name=$1
+                bam=$2
+                reference_fasta=$3  #assumes that there's a .genome file next to the reference .fa
+                genome_file = `basename $3 .fa`
```

Comment:
It says that it's looking for a `.genome` file, but where does the `.genome` part come from?  `all_sequences.fa` becomes `all_sequences` with the `basename` call, right?

Reply:
probably best just to require it as an input, (as we do in the bisulfite pipeline, IIRC)